recipe/meta.yaml

```diff
@@ -44,3 +44,4 @@ about:
 extra:
   recipe-maintainers:
     - marcelotrevisani
+    - xylar
```

Comment:
@marcelotrevisani, I certainly don't need to be a maintainer if you'd rather do this on your own.  Just helping out if you need it.